### DISCUSSION TITLE
addDoubletScores

### DISCUSSION THREAD

**Discussion by pengxin2019** [GreenleafLab/ArchR#1358] [Questions / Documentation]
Hi there,
I recently noticed that the `geneAnnotation` object I have used for `createArrowFiles` is not as good as I expected since the number of transcript in the `TxDb.Sscrofa.UCSC.susScr11.refGene` is much less than I thought. I am wondering which information is based on for the `addDoubletScores` function. Is it based on the fragment/peaks  information or the **genescore** (transcript or gene) information? I read through the full tutorial  of ArchR and did not see this info. If it is based on the fragment/peaks, I can still use the detected doublet result at least.

Thanks

**Comment by rcorces** [comment]
By default, doublet scores are based on fragments (tiles) information because `useMatrix = "TileMatrix"` is the functions default.

![image](https://user-images.githubusercontent.com/10343046/160289727-5ad20584-a791-43b2-a4e2-64ed8af56dbe.png)
